corehq/apps/preview_app/templates/preview_app/base.html

```diff
@@ -0,0 +1,150 @@
+{% load hq_shared_tags %}
```

Comment:
We could package some of the bits up in a partial, but this may count as significantly different to not do that. The base has a lot of extra bloat that we don't need to run Formplayer and also includes the header and footer which shouldn't be a part of it. I'll try and see if there's a good way to keep dependencies in sync with real cloudcare because that has been a huge source of pain in edit forms. That page has broken so many times because we add dep to cloudcare and forget to add it to edit forms